web-server/plugins/slycat-parameter-image/js/ui.js

```diff
@@ -1943,7 +1959,7 @@ $(document).ready(function () {
       // and we don't want long calls to come back now and filter anything.
       if (filterxhr) {
         filterxhr.abort();
-        console.debug("filter xhr aborted because last filter was closed");
+        // console.debug("filter xhr aborted because last filter was closed");
```

Comment:
delete

Reply:
Done in ec52d4f99280f8fd20af79b15c75b405a7fc2f9a